fhir-search/src/main/java/com/ibm/fhir/search/SearchConstants.java

```diff
@@ -41,25 +41,13 @@ private SearchConstants() {
 
     public static final String LOG_BOUNDARY = "---------------------------------------------------------";
 
-    // XML Processing.
-    public static final String DTM_MANAGER = "com.sun.org.apache.xml.internal.dtm.DTMManager";
-
-    public static final String DTM_MANAGER_DEFAULT = "com.sun.org.apache.xml.internal.dtm.ref.DTMManagerDefault";
```

Comment:
These were originally here for a performance reason... not sure why we dropped the use, but suspect we left from DSTU2 to R4 Migration.

Reply:
we still have this line in our server's jvm.options file:
```
-Dorg.apache.xml.dtm.DTMManager=org.apache.xml.dtm.ref.DTMManagerDefault
```

so yeah, not sure how these constants were used in past, but i think they are safe to remove now